study.md

```diff
@@ -100,26 +102,30 @@ pairs.
 | occupation | 'propane and propane accessories salesman' |
 
 ```javascript
-// your answer here
-```
 
-## Adding Dictionary Properties in JavaScript
+let hank = {
+  family_name: "hill",
+  given-name: "hank",
```

Comment:
This key needs to be wrapped in quotation marks because it has a special character in it.  `'given-name': "hank",`.